wasswa-derick/NairobiJavaGeeks
# [Chore #155738450] Circle CI & Code Coverage Integration

#### What does this PR do?
* Circle CI & Code Coverage Integration

#### Description of Task to be completed?
* Integrate Circle CI builds to the project with a status badge as well.
* Integrate Code Coverage status

#### How should this be manually tested?
* In your browser, enter ```https://github.com/wasswa-derick/NairobiJavaGeeks``` in the URL field.
* In the project ReadMe, there are status badges, check their status and click to navigate for more information.

#### What are the relevant pivotal tracker stories?
[#155738450](https://www.pivotaltracker.com/story/show/155738450)

#### Screenshots (if appropriate)

* **Status Badges**

<img width="1096" alt="badgese" src="https://user-images.githubusercontent.com/39955231/45755713-c898be80-bc27-11e8-9ed3-e0cfcde4e869.png">
